Assets/Scripts/Managers/DeliveryManager.cs

```diff
@@ -26,7 +26,7 @@ public class DeliveryManager : MonoBehaviour
     // The reason we allow for maxBoxesToDeliver to be specified, is so if we want
     // we could require only 1 meat, but specify the number of boxes that show up to be 2
     // This way, we don't have to flood the player with boxes early in the game
-    public bool DeliverMeat(List<Constants.Meats> requiredMeats, int maxBoxesToDeliver=Constants.MAX_MEAT_DELIVERY_SPOTS)
+    public bool DeliverMeat(List<PizzaOrder.MeatItem> requiredMeats, int maxBoxesToDeliver=Constants.MAX_MEAT_DELIVERY_SPOTS)
```

Comment:
Side note, should this be a `HashSet`? This is a unique list right?

Reply:
yep yep - you right!